compiler/control/OMROptions.cpp

```diff
@@ -2405,9 +2405,8 @@ OMR::Options::processOptionsJIT(char *jitOptions, void *feBase, TR_FrontEnd *fe)
       _jitCmdLineOptions = new (PERSISTENT_NEW) TR::Options();
       _cmdLineOptions = _jitCmdLineOptions;
       }
-
-   if (_jitCmdLineOptions)
-      memset(_jitCmdLineOptions, 0, sizeof(TR::Options));
+   else
+      _jitCmdLineOptions->init();
```

Comment:
Maybe you meant to do
```
if (_jitCmdLineOptions)
   _jitCmdLineOptions->init();
```
?

Reply:
We only want to call `init()` if the constructor hasn't already been used to create and initialize a new `TR::Options` object, so I think we actually have to use `else` instead of `if (_jitCmdLineOptions)`

Reply:
Well, there's two problems:
1. You removed the `if` branch so the code won't compile at the moment.
2. The else means `_jitCmdLineOptions` is NULL, in which case calling `init()` will result in a seg fault.

Reply:
Oh nvm, I didn't realize the `else` now belongs to the previous `if` above.